src/frontend_workspaces/frontend/src/carbon-chat/CarbonChat.css

```
@@ -434,3 +434,250 @@
+  font-size: 12px;
+}
+
+.cuga-slash-skill-chip__details dd {
+  margin: 0;
+}
+
+.cuga-slash-skill-chip__details code {
+  font-family: ui-monospace, SFMono-Regular, Menlo, monospace;
+  font-size: 12px;
+  color: #1f2328;
+  word-break: break-word;
```

Stylelint (17.11.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)